src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -1448,6 +1532,16 @@ private bool HasBindingMarkupExtension(XamlMemberDefinition valueNode)
 				);
 		}
 
+		private bool HasXBindMarkupExtension(XamlObjectDefinition objectDefinition)
+			=> objectDefinition
+				.Members
+				.Any(o => o.Objects.Any(o => o.Type.Name == "Bind"));
+
+		private bool HasResourceMarkupExtension(XamlObjectDefinition objectDefinition)
+			=> objectDefinition
+				.Members
+				.Any(o => o.Objects.Any(o => o.Type.Name == "Bind" || o.Type.Name == "StaticResource" || o.Type.Name == "ThemeResource"));
```

Comment:
Is the `o.Type.Name == "Bind"` intended here?

Reply:
Yes it is. It's supposed to be used for "component" backing field generation, but the name of the method is not appropriate. I'll update the name.